package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bigcommerce/stencil-cli",
-  "version": "1.15.4",
+  "version": "1.15.5",
```

Comment:
Can we not bump the release as part of the PR. Let's do it after we merge so we can test the feature as well.